### DISCUSSION TITLE
Roadmap to Stable v1.0.0?

### DISCUSSION THREAD

**Discussion by jack-evans** [hibiken/asynq#481] [Q&A] [answered]
Is there a roadmap to marking this project as stable I.e. v1.0.0? 

Really want to use it in my teams code base but APIs under heavy active development make me nervous 😄

**Comment by hibiken** [comment, marked as answer]
@jack-evans Thank you for the question.
Related issue: #242

At this point, the existing package APIs are quite stable. 
One of the main reason that I haven't committed to v1.0 is because I'm planning to implement a Workflow feature (#244) which may need to iterate on to land on a good API. 
Whenever there's a change in API, I always make sure to document it in the release note and changelog. 

Let me know if that addresses your concerns!
